src/validatedODEs.jl

```diff
@@ -51,8 +51,9 @@ function remainder_taylorstep!(f!::Function, t::Taylor1{T},
         Δx = Δ
     end
 
-    # If it didn't work, throw an error
-    @format full
+    # If it didn't work and adaptive is false, throw an error
+    adaptive && return (false, Δx, t[0])
```

Comment:
The comment does not match.

Reply:
@schillic if the program is outside the loop it means that "it didn't work", that the program couldn't prove the contraction property. then there are two options: either give up (if `adaptive` is false), or try again with another tolerance (`adaptive` is true). the first element in the tuple `(false, ...)` is a flag to say whether picard iteration succeeded or not.

Reply:
Oh sorry, I see now: the error comes below :sweat_smile:

Reply:
Related to this comment: Shall we throw an error (and break the integration) or rather a warning, and (in the main function) stop continuing the integration due to this type of failures?

Reply:
(The same would apply in other parts of the code.)